content/02.introduction.md

```diff
@@ -9,12 +9,20 @@ The low survival rates for some pediatric tumors are clearly multifactorial, exp
-This is, in part, due to pharmaceutical company priorities and/or concerns regarding toxicity leading, making it challenging for researchers to obtain to new therapeutic agents for pediatric clinical trials.
-Critically, as of August 18, 2020, an amendment to the Pediatric Research Equity Act called the "Research to Accelerate Cures and Equity (RACE) for Children Act" mandates that all new adult oncology drugs also be tested in children when the molecular targets are relevant to a particular childhood cancer. 
-This regulatory change is poised to accelerate identification of novel and effective therapeutic for pediatric diseases that have otherwise been overlooked.
+To leverage diverse scientific and analytical expertise to analyze the PBTA data, we created an open science model and incorporated features, such as analytical code review [@doi:10.1093/aje/kwab092; @doi:10.7287/peerj.preprints.3210v1] and continuous integration to test data and code [@doi:10.1038/nbt.3780; @doi:10.7287/peerj.preprints.3210v1], to improve reproducibility throughout the life cycle of our project, termed OpenPBTA.
+We anticipated that a model of open collaboration would enhance the value of our effort to the pediatric brain tumor research community. 
+Openly sharing data and code in real time allows others to build upon our work more rapidly, and publications that include data and code sharing may have a greater impact [@doi:10.1371/journal.pone.0000308; @doi:10.1371/journal.pcbi.1008867].
 Here, we present a comprehensive, collaborative, open genomic analysis of nearly 2,000 tumor specimens and 38 cell lines, comprised of 59 distinct brain tumor histologies from 943 patients. 
-These analyses can be used to support the RACE Act in rational clinical trial design to include children.
+These analyses can be used to support the RACE Act in rational clinical trial design to include children with brain tumors.
```

Comment:
I wanted to make this sentence a little more explicit. It would not surprise me if a PR reviewer says, "TMB calculation code, etc. is not limited to brain tumors." This change reads better in my opinion.